workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -65,6 +74,43 @@ describe('WBCGroupManagemntPlugin', () => {
     ddbMock.reset();
   });
 
+  describe('createGroup', () => {
+    it('returns `created` as true when the group was successfully created', async () => {
+      const groupId = 'groupId';
+
+      const response = await wbcGroupManagementPlugin.createGroup({ groupId, authenticatedUser: mockUser });
+
+      expect(response).toMatchObject<CreateGroupResponse>({ data: { groupId } });
+    });
+
+    it('throws IdpUnavailableError when the IdP encounters an error', async () => {
+      const groupId = 'groupId';
+      mockUserManagementPlugin.createRole = jest.fn().mockRejectedValue(new IdpUnavailableError());
+
+      await expect(
+        wbcGroupManagementPlugin.createGroup({ groupId, authenticatedUser: mockUser })
+      ).rejects.toThrow(IdpUnavailableError);
+    });
+
+    it('throws PluginConfigurationError when the UserManagementService has a configuration error', async () => {
+      const groupId = 'groupId';
+      mockUserManagementPlugin.createRole = jest.fn().mockRejectedValue(new PluginConfigurationError());
+
+      await expect(
+        wbcGroupManagementPlugin.createGroup({ groupId, authenticatedUser: mockUser })
+      ).rejects.toThrow(PluginConfigurationError);
+    });
+
+    it('throws GroupAlreadyExistsError when the group already exists', async () => {
```

Comment:
should this be `when the role already exists` instead of `group`?

Reply:
for dynamic authorization `group` is used, for user management `role` is used. Internally a `RoleAlreadyExistsError` is thrown, but it gets turned into a `GroupAlreadyExistsError` to match the dynamic authz naming conventions